vm/sut.go

```diff
@@ -443,25 +439,25 @@ func (s *SUT) EmptyDisk(disk string) {
 	_, _ = s.Command("sleep 5")
 }
 
-// SetCOSCDLocation gets the location of the iso attached to the vbox vm and stores it for later remount
-func (s *SUT) SetCOSCDLocation() {
+// SetCDLocation gets the location of the iso attached to the vbox vm and stores it for later remount
```

Comment:
I am even wondering if we should drop all this or not... this is VBox coupled and for the BIOS firmware use case. I doubt this is actually being used in any test in current `main` branch....

Reply:
mmm weird, in elemental-toolkit I think we were no longer using VBox, as it does not properly support EFI. All this CD commands and code was to handle booting from BIOS, to _eject_ CD, so after installing it was not rebooting on the CD again.